elm-package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "1.1.7",
```

Comment:
Please do not bump the version when submitting a package. That's for the maintainer (me in this case) to do when releasing. Besides, this isn't the correct new version.